middleware/redirects.ts

```diff
@@ -1,16 +1,28 @@
 import { Context } from '@nuxt/types'
 
+import { questionsEngToPl } from '@/constants'
+
 export default function ({ route, redirect }: Context) {
-  if (route.fullPath.match(/^\/categories/)) {
+  if (route.path.match(/^\/categories/)) {
     redirect(301, '/kategorie')
   }
 
-  if (route.fullPath.match(/^\/login/)) {
+  if (route.path.match(/^\/login/)) {
     redirect(301, '/logowanie')
   }
 
-  // TODO handle query params
-  if (route.fullPath.match(/^\/questions/)) {
-    redirect(301, '/pytania')
+  if (route.path.match(/^\/questions/)) {
+    let params = route.fullPath.split('?')[1]
+
+    if (!params) return redirect(301, '/pytania')
+
+    for (const paramOrOption of Object.keys(questionsEngToPl)) {
```

Comment:
Nieczytelna funkcja, 
wydziel do prywatnej metody która jako parametr ma `...rest` i podaj jej oba enumy.
W środku zrób `foreach` loopa który będzie brał po enumie i loopował się `for(let item of Object.keys(entry))` na wartościach tego enuma. Zrób obliczenia i zwróć wynik

Reply:
> która jako parametr ma ...rest

Czy to będzie czytelne? Jeśli wiadomo z ilu argumentów funkcja będzie korzystać, to w jakim celu używać tutaj _rest parameters_, zamiast zwyczajnie je przekazać? Chyba, że czegoś nie rozumiem.